test_scripts/API/PerformAudioPassThru/ideas_PAPT_new_param_audioPassThruIcon.txt

```diff
@@ -0,0 +1,56 @@
+  +./test_scripts/API/PerformAudioPassThru/001_ATF_P_PerformAudioPassThru_SUCCESS_All_Params.lua 
+  Check_that_SUCCESS_returned_when_all_parameters_valid_within_bounds
+  +./test_scripts/API/PerformAudioPassThru/002_ATF_P_PerformAudioPassThru_SUCCESS_All_Params_Upper.lua
+  Check_that_SUCCESS_returned_when_all_param_valid_upper_bound 
+  +./test_scripts/API/PerformAudioPassThru/003_ATF_P_PerformAudioPassThru_SUCCESS_All_Params_Lower.lua
+  Check_that_SUCCESS_returned_when_all_param_valid_lower_bound  
+  +./test_scripts/API/PerformAudioPassThru/004_ATF_P_PerformAudioPassThru_SUCCESS_Only_Mandatory_Params_with_audioPassThruIcon
+  Check_that_SUCCESS_returned_when_only_mandatory_param_and_audioPassThruIcon_sent
+  +./test_scripts/API/PerformAudioPassThru/005_ATF_P_PerformAudioPassThru_WARNINGS_Different_Speech_Capabilities_PRE_RECORDED.lua
+  Check_that_WARNINGS_returned_when_speech_capabilities_PRE_RECORDED_and_audioPassThruIcon_sent
+  +./test_scripts/API/PerformAudioPassThru/006_ATF_P_PerformAudioPassThru_WARNINGS_Different_Speech_Capabilities_SAPI_PHONEMES.lua 
+  Check_that_WARNINGS_returned_when_speech_capabilities_SAPI_PHONEME_and_audioPassThruIcon_sent
+  +./test_scripts/API/PerformAudioPassThru/007_ATF_P_PerformAudioPassThru_WARNINGS_Different_Speech_Capabilities_LHPLUS_PHONEMES.lua 
+  Check_that_WARNINGS_returned_when_speech_capabilities_LHPLUS_PHONEMES_and_audioPassThruIcon_sent
+  +./test_scripts/API/PerformAudioPassThru/008_ATF_P_PerformAudioPassThru_WARNINGS_Different_Speech_Capabilities_SILENCE.lua 
+  Check_that_WARNINGS_returned_when_speech_capabilities_SILENCE_and_audioPassThruIcon_sent
+  +./test_scripts/API/PerformAudioPassThru/009_ATF_P_PerformAudioPassThru_SUCCESS_Dynamic_Image_All_Params.lua
+  Check_that_SUCCESS_returned_when_image_type_DYNAMIC_All_Params_and_audioPassThruIcon_sent
+  +./test_scripts/API/PerformAudioPassThru/010_ATF_P_PerformAudioPassThru_SUCCESS_Dynamic_Image_MandatoryParams_and_audioPassThruIcon.lua
+  Check_that_SUCCESS_returned_when_image_type_Dynamic_Mandatory_params_and_audioPassThruIcon
+  +./test_scripts/API/PerformAudioPassThru/011_ATF_P_PerformAudioPassThru_WARNINGS_Valid_Request_Without_audioPassThruIcon.lua
+  Check_that_WARNINGS_returned_Without_audioPassThruIcon_all_params
+  +./test_scripts/API/PerformAudioPassThru/012_ATF_P_PerformAudioPassThru_WARNINGS_Mandatory_Only_Without_audioPassThruIcon.lua
+  Check_that_WARNINGS_returned_Without_audioPassThruIcon_mandatory_params
+  +./test_scripts/API/PerformAudioPassThru/013_ATF_N_PerformAudioPassThru_INVALID_DATA_All_Params_audioPassThruIcon_STATIC.lua 
+  Check_that_INVALID_DATA_returned_when_audioPassThruIcon_has_invalid_value_parameter_all_other_params_sent_and_valid_STATIC_image_type 

```

Comment:
@maria-mih, please add comment that script is performed in loop for invalid_data. 
Please add this for scripts 014-018